specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -13,14 +13,88 @@
+processes from derivation to execution without any additional developments of ZK circuit.
+
+ZKFDG implements the OP Stack's `IDisputeGame`, implying that it is fully compatible with the OP Stack's dispute game.
+This also implies that it can be applied to the OP Stack's multi-proof system in the future.
+
 ## ZK Fault Dispute Game Creation
 
+The ZK Fault Dispute Game is created by a validator selected by the
+[`ValidatorManager`](../../protocol/validator-v2/validator-manager.md) at intervals defined by the
+`SUBMISSION_INTERVAL` configured in the [`L2OutputOracle`](../../protocol/validation.md#l2-output-oracle-smart-contract)
+contract. The validator can create a ZKFDG-type dispute game through the `DisputeGameFactory` with an extra data. The
+extra data is composed as follows:
+
+```text
+extra_data = current_l2_block_num ++ prev_game_proxy
+
+current_l2_block_num = uint256
+prev_game_proxy      = bytes32
+```
+
+When the ZKFDG is created by the validator, it is verified that the validator is eligible to submit a claim and the
+`prev_game_proxy` is valid ZKFDG contract. The `prev_game_proxy` is used as the starting output root for the dissection
+process to find the first disagreeing block. If all validations pass, the address of this ZKFDG contract is stored with
+current L2 block number on the L2OutputOracle contract. This will be used for the creation of the next ZKFDG.
+
 ## Challenge Creation
 
+As with Kroma's previous Fault Proof System, a challenge can be initiated by a validator who disagrees the submitted
+claim. The challenge process begins by submitting the initial segments by challenger.
+
+```solidity
+/**
+     * @notice Creates a challenge against an invalid claim.
+     *
+     * @param _l1BlockHash   The block hash of L1 at the time the output L2 block was created.
+     * @param _l1BlockNumber The block number of L1 with the specified L1 block hash.
+     * @param _segments      Array of the segment. A segment is the first output root of a specific range.
+     */
+    function createChallenge(
+        bytes32 _l1BlockHash,
+        uint256 _l1BlockNumber,
+        bytes32[] calldata _segments
+    ) external;
+```
+
 ## Dissection
 
+The dissection process is carried out in the same manner as in Kroma's previous Fault Proof System. Through interactions
+between the challenger and the defender (the game creator), the first disagreeing output root can be specified.
+
+```solidity
+    /**
+     * @notice Selects an invalid section and submit segments of that section.
+     *
+     * @param _challenger  Address of the challenger.
+     * @param _pos         Position of the last valid segment.
+     * @param _segments    Array of the segment. A segment is the first output root of a specific range.
+     */
+    function dissect(
+        address _challenger,
+        uint256 _pos,
+        bytes32[] calldata _segments
+    ) external;
+```
+
 ## Proving Fault using ZKVM
```

Comment:
Maybe here some contents will be added, as mentioned in [this comment](https://github.com/kroma-network/kroma-specs/pull/19#pullrequestreview-2268055565), right?

Reply:
Yes you're right

Reply:
done in #21